### PR TITLE
Extended form section from jsonForm object to allow empty fields

### PR DESCRIPTION
**Pull request by joroca**

### CODE REVIEW

File: gecoscc/templates/base_tree.jinja2 [hunk 1]
```diff
@@ -127,24 +127,6 @@
 
     <div id="staging-modal-viewport"></div>
     <div id="staging-modal-changes"></div>
-
```
Comment (alfem) [inline gecoscc/templates/base_tree.jinja2]: Why are you deleting thsse lines?

File: gecoscc/templates/base_tree.jinja2 [hunk 2]
```diff
@@ -163,7 +145,8 @@
     <script src="{{request.application_url}}/static/js/libs/jquery.dataTables-1.9.4.min.js"></script>
     <script src="{{request.application_url}}/static/js/libs/DT_bootstrap.min.js"></script>
     <script src="{{request.application_url}}/static/js/libs/chosen.jquery-0.13.0.min.js"></script>
-    <script src="{{request.application_url}}/static/js/libs/jsonform-b3-1.0.0.js"></script>
```
Comment (alfem) [inline gecoscc/templates/base_tree.jinja2]: As we are changing the original jsonform & jsonpath, I suggest to change file names to something like jsonform-b3-1.0.0.gecos1.js & jsonpath-0.8.0.gecos1.js

File: gecoscc/templates/settings.jinja2 [hunk 3]
```diff
@@ -70,10 +70,10 @@
                       }
                     }
                 }
-                {% elif setting.type == "Mimetypes" %}
```
Comment (alfem) [inline gecoscc/templates/settings.jinja2]: These fields have been renamed, and you are not pulling latests changes. Please, update your local copy before a new Pull Request.

File: gecoscc/templates/base_tree.jinja2 (hunk 1 above)
Comment (joroca) [inline gecoscc/templates/base_tree.jinja2]: Really, the code is missing is coming under the two lines above. Updated.

File: gecoscc/templates/settings.jinja2 (hunk 3 above)
Comment (joroca) [inline gecoscc/templates/settings.jinja2]: Done.

File: gecoscc/templates/base_tree.jinja2 (hunk 2 above)
Comment (joroca) [inline gecoscc/templates/base_tree.jinja2]: Done.

**Pull request merged**